File: sort/Bubble_Sort.py
def bubble(arr):
    l=len(arr)
    for i in range(l):
        for j in range(i-1):
            if arr[j]>arr[j+1]:
                temp=arr[j]
                arr[j]=arr[j+1]
                arr[j+1]=temp
    return arr

def func():
    ls=list(map(int,input("Enter elements:").split()))
    ls=bubble(ls)
    print(ls)

if __name__=='__main__':
    func()

File: sort/bubble_Sort.cpp
#include<iostream>
using namespace std;
int bubble(int arr[],int n)
{
    int temp;
    for(int i=0;i<n;i++)
    {
        for(int j=0;j<n-1;j++)
        {
            if(arr[j]>arr[j+1])
            {
                    temp=arr[j];
                    arr[j]=arr[j+1];
                    arr[j+1]=temp;
            }
        }
    }
    return *arr;
}
int main()
{
    int n;
    cout<<"Enter number of elements:";
    cin>>n;
    int arr[n];
    cout<<"Enter the elements:";
    for(int i=0;i<n;i++)
    {
        cin>>arr[i];
    }
    *arr=bubble(arr,n);
    for(int i=0;i<n;i++)
    {
        cout<<arr[i]<<' ';
    }
}
